### DISCUSSION TITLE
Re-locking in fast PID only mode

### DISCUSSION THREAD

**Discussion by Samuel-Gebe** [linien-org/linien#381] [Q&A] [answered]
Hello, is it possible to have the unlock and auto re-lock features in v 0.3.2 without having to modulate/demodulate with the red pitaya? Also what is the bandwidth of the feedback loop when not in PID only mode? I am curious because the PID only mode was introduced in [v0.5.0-1](https://github.com/linien-org/linien/releases/tag/v0.5.0-1) without having the re-locking feature. Any information would be greatly appreciated.

**Comment by bleykauf** [comment, marked as answer]
I am not sure how it was handled in that version but you could try to set modulation frequency and/or amplitude to 0. At least in later versions this circumvents the modulation/demodulation step.

In fast-mode (not available before 0.5.0-1, the control bandwidth is almost 1 MHz, for the standard mode (with modulation/demodulation turned off as above)  you should expect something like 300 kHz.
